docs/examples/buildConfigs/README.md

```diff
@@ -0,0 +1,28 @@
+# BuildConfig
+
+A build configuration describes a single build definition and a set of triggers for when a new build should be created. A build configuration is defined by a BuildConfig.
+
+In Kedge, defining BuildConfigs has been implemented by merging `BuildConfigSpec` and `ObjectMeta` at the root YAML level of the spec.
```

Comment:
`defining BuildConfigs are implemented`

Reply:
should it not be - `defining BuildConfigs is implemented`?